### ISSUE TITLE
Sign shop no longer available? 

### ISSUE THREAD

**Issue by donnie8986** [Shampaggon/CrackShot#15]
Hello, 

I recall about two years or so back having made a Crackshot shop using just Crackshot's built in sign shop. I tried looking into the config files, but I no longer see options to configure shops. Was this feature taken out? If so, is there another way to create shops for this plugin? Thanks in advance. 

Donnie

**Comment by noahcoetsee** [comment]
Yes, I've noticed that the CrackShot shops are no longer in effect. Use this instead: https://www.spigotmc.org/resources/signshop.10997/

Specifically, to reflect the features of the CrackShot Shops: I use [iBuy] Signs.
